src/site/content/en/blog/building-a-breadcrumbs-component/index.md

```diff
@@ -355,7 +356,7 @@ maintaining the built-in functionality.
   inline-size: 100%;
   block-size: 100%;
   opacity: .01;
-  font-size: 16px; /* fix iOS zoom */
+  font-size: 100% /* Defaults to 16px; fixes iOS zoom */
```

Comment:
```suggestion
  font-size: min(100%, 16px); /* Defaults to 16px; fixes iOS zoom */
```

thoughts on this? leverage `min()` to create a more adaptive select input size that won't break if someone changes the document font size?

Reply:
Even better! `<3`